docs/docs/command-line-interface.mdx

```diff
@@ -214,6 +214,13 @@ at http://localhost:5005/visualization.html
 as soon as the session has started. This diagram can take some time to generate.
 To skip the visualization, run `rasa interactive --skip-visualization`.
 
+:::info Add the `assistant_id` key introduced in 3.5
+
+Running interactive learning with a pre-trained model whose metadata does not include the `assistant_id`
+will exit with an error. If this happens, add the required key with a unique identifier value and re-run training.
```

Comment:
We should state where it should be added.

Reply:
➕